src/cryptography/convert.js

```diff
@@ -91,14 +91,10 @@ export const parseEncryptedPassphrase = encryptedPassphrase => {
 		throw new Error('Encrypted passphrase to parse must be a string.');
 	}
 	const keyValuePairs = querystring.parse(encryptedPassphrase);
-	const { salt, cipherText, iv, tag, version } = keyValuePairs;
-	const iterations =
-		keyValuePairs.iterations !== undefined
-			? parseInt(keyValuePairs.iterations, 10)
-			: null;
+	const { iterations, salt, cipherText, iv, tag, version } = keyValuePairs;
 
 	return {
-		iterations,
+		iterations: iterations === undefined ? undefined : parseInt(iterations, 10),
```

Comment:
What is the reason of strictly checking only `undefined`?

Reply:
If it's present it should be a numerical string (therefore `parseInt`able).